release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -474,9 +473,72 @@ The terminology for the `ip_whitelist` and `ip_blacklist` annotations have been
+
+[id="ocp-4-18-release-notes-storage-secret-csi-driver_{context}"]
+==== Secret Store CSI Driver Operator is generally available
+The Secrets Store Container Storage Interface (CSI) Driver Operator, `secrets-store.csi.k8s.io`, allows {product-title} to mount multiple secrets, keys, and certificates stored in enterprise-grade external secrets stores into pods as an inline ephemeral volume. The Secrets Store CSI Driver Operator communicates with the provider using gRPC to fetch the mount contents from the specified external secrets store. After the volume is attached, the data in it is mounted into the container’s file system. The Secrets Store CSI Driver Operator was available in {product-title} 4.14 as a Technology Preview feature. {product-title} 4.18 introduces this feature as generally available. For more information about the Secrets Store CSI driver, see xref:../storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc#persistent-storage-csi-secrets-store[Secrets Store CSI Driver Operator].
+
+// For information about using the Secrets Store CSI Driver Operator to mount secrets from an external secrets store to a CSI volume, see xref:../nodes/index#nodes-pods-secrets-store[Providing sensitive data to pods by using an external secrets store].
+
+[id="ocp-4-18-release-notes-storage-pv-last-phase-transition-time_{context}"]
+==== Persistent volume last phase transition time parameter is generally available
+{product-title} 4.16 introduced a new parameter, `LastPhaseTransitionTime`, which has a timestamp that is updated every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`). For {product-title} 4.18, this feature is generally available.
+
+For more information about using the persistent volume last phase transition time parameter, see xref:../storage/understanding-persistent-storage.adoc#persistent-volumes_understanding-persistent-storage[Persistent volumes].
+
+[id="ocp-4-18-release-notes-storage-volume-multi-vcenter-support_{context}"]
+==== Multiple vCenter support for vSphere CSI is generally available
+{product-title} 4.17 introduced the ability to deploy {product-title} across multiple vSphere clusters (vCenters) as a Technology Preview feature. In {product-title} 4.18, Multiple vCenter support is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-18-release-notes-storage-always-honor-reclaim-policy_{context}"]
+==== Always apply persistent volume reclaim policy (Technical Preview)
```

Comment:
```suggestion
==== Always honor persistent volume reclaim policy (Technical Preview)
```

Reply:
In case to keep title consistent with Upstream feature

Reply:
Done.